### ISSUE TITLE
Neovim (vimR) throws error when `has('python3') == 1`

### ISSUE THREAD

**Issue by dNitro** [maralla/completor.vim#80]
When python 3 is installed ( checked with `has('python3')` and it returns 1 ), completor throws the following error and no omni-completion:
```bash
Error detected while processing function <SNR>90_complete[7]..completor#do_complete[2]..completor#trigger[6]..completor#utils#get_completions[1]..provider#python3#Call:
line   18:
Traceback (most recent call last):

  File "<string>", line 1, in <module>

  File "/Users/david/.vim/plugged/completor.vim/pythonx/completor/api.py", line 14, in wrapper
    return func(vim.bindeval('a:'))

  File "/Users/david/.vim/plugged/completor.vim/pythonx/completor/api.py", line 28, in get_completions
    return c.get_completions(args['msg']) if c else []

  File "/Users/david/.vim/plugged/completor.vim/pythonx/completor/__init__.py", line 141, in get_completions
    return self.parse(base)

  File "/Users/david/.vim/plugged/completor.vim/pythonx/completers/common/omni.py", line 61, in parse
    return omnifunc(0, to_bytes(base, get_encoding())[codepoint:])

  File "/Users/david/.vim/plugged/completor.vim/pythonx/completor/patch.py", line 30, in inner
    return _bytes(ret)

  File "/Users/david/.vim/plugged/completor.vim/pythonx/completor/patch.py", line 11, in _bytes
    data[i] = _bytes(e)

  File "/Users/david/.vim/plugged/completor.vim/pythonx/completor/patch.py", line 13, in _bytes
Error detected while processing function <SNR>90_complete[7]..completor#do_complete[2]..completor#trigger[6]..completor#utils#get_completions[1]..provider#python3#Call:
line   18:
    for k, v
```
Whithout python3 everything works fine but if **completions include non-ascii charachters** it also throws encoding error with python2.

So there is two issues:
 - The above error when using python 3
 - Encoding error when using python 2

**vimR** `:version` returns:
```bash
NVIM 0.2.0-dev
Build type: Release
Compilation:
  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
  -Wconversion
  -U_FORTIFY_SOURCE
  -D_FORTIFY_SOURCE=1
  -O2
  -DNDEBUG
  -DDISABLE_LOG
  -Wall
  -Wextra
  -pedantic
  -Wno-unused-parameter
  -Wstrict-prototypes
  -std=gnu99
  -Wvla
  -fstack-protector-strong
  -fdiagnostics-color=auto
  -DINCLUDE_GENERATED_DECLARATIONS
  -DHAVE_CONFIG_H
  -I/Users/hat/.jenkins/workspace/vimr_build/neovim/build/config
  -I/Users/hat/.jenkins/workspace/vimr_build/neovim/src
  -I/Users/hat/.jenkins/workspace/vimr_build/neovim/.deps/usr/include
  -I/Users/hat/.jenkins/workspace/vimr_build/neovim/.deps/usr/include
  -I/Users/hat/.jenkins/workspace/vimr_build/neovim/.deps/usr/include
  -I/Users/hat/.jenkins/workspace/vimr_build/neovim/.deps/usr/include
  -I/usr/local/opt/gettext/include
  -I/usr/include
  -I/Users/hat/.jenkins/workspace/vimr_build/neovim/build/src/nvim/auto
  -I/Users/hat/.jenkins/workspace/vimr_build/neovim/build/include
Compiled by hat@smiley.fritz.box

Optional features included (+) or not (-): +acl   +iconv    +jemalloc -tui      
For differences from Vim, see :help vim-differences

   system vimrc file: "$VIM/sysinit.vim"
   fall-back for $VIM: "/usr/local/share/nvim"

```

**Comment by maralla** [comment]
Here is the fix #96 for this problem. Please update and try it again.

**Comment by dNitro** [comment]
Unfortunately I does not have access to my mac machine for a while so presume it resolved. Many Thanks.
